app/views/activities/_filters.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="activity-filters ">
+  <%= form_with url: project_activities_path(current_project.id), method: :get, remote: true, class: 'form-row d-flex flex-row' do |f| %>
+    <%= f.select :user, options_from_collection_for_select(User.all, :id, :name, params[:user]), { include_blank: 'Filter by user'}, { class: 'form-select m-2 form-select-sm', onchange: '$(this.form).submit();' } %>
+    <%= f.select :type, Activity.select(:trackbable_type).distinct.pluck(:trackable_type), { include_blank: 'Filter by type', selected: params[:type]}, { class: 'form-select m-2 form-select-sm', onchange: '$(this.form).submit();' } %>
+
+    <div id="" class="input-group">
```

Comment:
why do we need this blank ID here?

Reply:
we dont. it's a leftover from the previous implementation with the external library datepicker. i removed it. thanks for noticing.